app/models/answer.rb

```diff
@@ -1,5 +1,7 @@
 class Answer < ActiveRecord::Base
   include VotesCounter
+  include ReputationConcern
```

Comment:
I think you are right. And then I will do `User.update_user_reputation` from both the `Vote.process_vote` and `Answer.accept` :bulb: